itou/www/employee_record_views/views.py

```diff
@@ -105,6 +97,22 @@ def get_prefix(self, request, *args, **kwargs):
         """
         return f"company_{self.company.pk}_add_employee_record"
 
+    def check_wizard_state(self, *args, **kwargs):
+        step_url = kwargs.get("step", None)
+        if step_url == self.steps.last and self.steps.current == self.steps.first:
```

Comment:
On pourrais pas juste faire ça ?
```suggestion
        if step_url != self.steps.current:
```

Quoique j'ai l'impression que le `step` n'est pas encore mis à ce moment car ça l'air d'être fait dans `NamedUrlWizardView.get()` :/.

Reply:
Ça plante pour l'étape done.
Donc il faut ajouter `step_url != self.steps.current and step_url != "done"`
mais c'est une bonne idée, ça force à l'étape en cours, c'est plus simple.

Reply:
(et les steps sont créés dans le dispatch, donc c'est bon)